BREEZE.rst

```diff
@@ -348,18 +349,43 @@ In case of disk space errors on macOS, increase the disk space available for Doc
 Building the Images
 -------------------
 
-You can manually trigger building the local images using the script:
+You can manually trigger building the local CI image using this command:
 
 .. code-block::
 
-  ./breeze build-only
+  ./breeze build-ci-image
 
-The scripts that build the images are optimized to minimize the time needed to rebuild the image when
+And production image using this command:
+
+.. code-block::
+
+  ./breeze build-prod-image
+
+
+The images are build with default extras - different for CI image and different for production image
+and using python version chosen in breeze. You can change those parameters by specifying them as
+command line options. You can see default extras used via  ``./breeze flags`` or in the syntax example
+below.
+
+For example if you want to build python 3.7 version of production image with
+all extras installed you should run this command:
+
+.. code-block::
+
+  ./breeze build-prod-image --python 3.7 --extras "all"
```

Comment:
Is it the only option? Will `docker build` work too?

Reply:
IT will work  in one combination and it is slower because it is multi-segmented image which cannot be cached directly. If you want to reflect the actual build command it should be something like 

```
docker build . --build-arg PYTHON_MAJOR_MINOR_VERSION=3.7 --build-arg AIRFLOW_EXTRAS="all" --cache-from apache/airflow:master-build-3.7 --cache-from apache/airflow:master-3.7 
```

Reply:
I think it's not worth explaining it.

Reply:
I will actually add it to the docs :)

Reply:
Yeah, I think it's worth adding a note. Not everyone is using breeze and we should not make breeze a requirement for building a docker image :)

Reply:
I added a lot of docs already. But wait for next push - I separated information about both CI and Production images to separate IMAGES.rst file. Bear with me!